acbuild/acbuild.go

```diff
@@ -140,7 +144,45 @@ GLOBAL OPTIONS:
 // terminator.
 func runWrapper(cf func(cmd *cobra.Command, args []string) (exit int)) func(cmd *cobra.Command, args []string) {
 	return func(cmd *cobra.Command, args []string) {
+		if aciToModify != "" {
+			contextualCommands := []string{"begin", "end", "abort"}
+			for _, cc := range contextualCommands {
+				command := strings.Split(cmd.Use, " ")[0]
```

Comment:
is that a thinly veiled dig at me? ;-)
I don't know why I keep un-learning the slice semantics, too many false memories..